components/kubeflow/katib-launcher/src/launch_experiment.py

```diff
@@ -41,6 +41,9 @@ def __init__(self, version="v1alpha3", client=None):
     super(Experiment, self).__init__(ExperimentGroup, ExperimentPlural, version, client)
 
   def is_expected_conditions(self, inst, expected_conditions):
+    trials_running_count = inst.get('status', {}).get('trialsRunning', 0)
```

Comment:
You could skip the default value of zero here and use: ```if trials_running_count``` since default value is None, and int >0 is evaluated to true.

Reply:
i got it.